pkg/pidfile/pidfile.go

```diff
@@ -33,7 +34,15 @@ func readPidFile() (uint64, error) {
 	}
 
 	pid := string(bytes.TrimSpace(data))
-	_, err = os.Stat(filepath.Join(option.Config.ProcFS, pid))
+	if runtime.GOOS != "windows" {
```

Comment:
I'm not sure, but maybe we should stick with _linux/_windows suffixes,
even thought it's small change I wonder we should have just single way
to separate linux and windows code .. just a hunch this has a potential
to turn bad in future ... thoughts? @kkourt @mtardy

Reply:
I have created a new function and unit test